Header header
MotorPosition[] motor_positions
bool isMoving
